index.html

```diff
@@ -18,13 +18,152 @@
             <div id="splash-info">
                 <img class="logo" src="assets/imgs/logo.png" alt="TartanHacks logo"/>
                 <h1>TartanHacks</h1>
-                <p>TartanHacks is an annual, student-run hackathon run by <a href="http://www.scottylabs.org/">Scottylabs</a>.</p>
+                <p>TartanHacks is an annual hackathon run by <a href="http://www.scottylabs.org/">Scottylabs</a>, a student organisation.</p>
```

Comment:
I don't mean to nitpick this to death, but I think it sounds a little better to say

"... a student organization __at CMU__."